currency_rate_update/README.rst

```diff
@@ -14,11 +14,14 @@ The reference rates are based on the regular daily query procedure between centr
 Configuration
 =============
 
-To configure the module, go to *Accounting > Configuration > Multi-currencies > Rate Auto-download* and create one or several services to download rates from the Internet.
+To configure the module:
 
-Then, go to the page *Accounting > Configuration > Settings* and, in the section *Multi Currencies*, make sure that the option *Automatic Currency Rates Download* is enabled.
+1. Go to *Invoicing > Configuration > Rate Auto-download* and create one or several services to download rates from the Internet.
```

Comment:
better use ´#. ´ for enumerated list (and in this case, no need empty line in the list)